docs/logto-cloud/tenant-member-management.mdx

```diff
@@ -0,0 +1,52 @@
+---
+id: tenant-member-management
+title: Tenant member management
+sidebar_label: Tenant member management
```

Comment:
nit: If they are the same as the H1 title in the main content, they can be omitted in the frontmatter